game-headed/build.gradle

```diff
@@ -41,25 +41,963 @@ jar {
+            'units/Puppet_States/factory_hit.png',
+            'units/Puppet_States/factory_it_hit.png',
+            'units/Puppet_States/factory_it.png',
+            'units/Puppet_States/factory.png',
+            'units/Puppet_States/fighter_jp.png',
+            'units/Puppet_States/fighter_lr_jp.png',
+            'units/Puppet_States/fighter_lr.png',
+            'units/Puppet_States/fighter.png',
+            'units/Puppet_States/halftrack.png',
+            'units/Puppet_States/harbour_disabled.png',
+            'units/Puppet_States/harbour_hit.png',
+            'units/Puppet_States/harbour.png',
+            'units/Puppet_States/infantry.png',
+            'units/Puppet_States/marine.png',
+            'units/Puppet_States/rockets.png',
+            'units/Puppet_States/rockets_r.png',
+            'units/Puppet_States/s_destroyer.png',
+            'units/Puppet_States/submarine.png',
+            'units/Puppet_States/submarine_ss.png',
+            'units/Puppet_States/transport.png',
+            'units/Russians/aaGun_old.png',
+            'units/Russians/aaGun.png',
+            'units/Russians/aaGun_r.png',
+            'units/Russians/airfield_disabled.png',
+            'units/Russians/airfield_hit.png',
+            'units/Russians/airfield.png',
+            'units/Russians/armour.png',
+            'units/Russians/artillery.png',
+            'units/Russians/battlecruiser.png',
+            'units/Russians/battleship_hit.png',
+            'units/Russians/battleship.png',
+            'units/Russians/bomber_hb.png',
+            'units/Russians/bomber_lr_hb.png',
+            'units/Russians/bomber_lr.png',
+            'units/Russians/bomber.png',
+            'units/Russians/bunker_disabled.png',
+            'units/Russians/bunker_hit.png',
+            'units/Russians/bunker.png',
+            'units/Russians/carrier.png',
+            'units/Russians/cruiser.png',
+            'units/Russians/destroyer.png',
+            'units/Russians/factory_hit.png',
+            'units/Russians/factory_it_hit.png',
+            'units/Russians/factory_it.png',
+            'units/Russians/factory.png',
+            'units/Russians/fighter_jp.png',
+            'units/Russians/fighter_lr_jp.png',
+            'units/Russians/fighter_lr.png',
+            'units/Russians/fighter.png',
+            'units/Russians/halftrack.png',
+            'units/Russians/harbour_disabled.png',
+            'units/Russians/harbour_hit.png',
+            'units/Russians/harbour.png',
+            'units/Russians/infantry.png',
+            'units/Russians/marine.png',
+            'units/Russians/rockets_old.png',
+            'units/Russians/rockets.png',
+            'units/Russians/rockets_r.png',
+            'units/Russians/s_destroyer.png',
+            'units/Russians/submarine.png',
+            'units/Russians/submarine_ss_old.png',
+            'units/Russians/submarine_ss.png',
+            'units/Russians/transport.png',
```

Comment:
I don't think it's the best idea to have all of the files hardcoded in `build.gradle`, this clearly blows this file up.
I'd suggest creating a dedicated "index" file that is downloaded by gradle from the assets repository (or stored locally if you'd prefer that option) just like all the files below and then run the download script for every line.

Haven't actually tested it, but something like this should work:
```groovy
download {
  src "https://raw.githubusercontent.com/triplea-game/assets/master/index.txt"
  dest "$projectDir/assets/index.txt"
  overwrite false
}
file('index.txt').eachLine { path ->
  download {
    src "https://raw.githubusercontent.com/triplea-game/assets/master/game_headed_assets/$path"
    dest "$projectDir/assets/$path"
    overwrite false
  }
}
```

if the file follows this format:
```
units/Russians/marine.png
units/Russians/rockets.png
```
etc.